app/controllers/comments_controller.rb

```diff
@@ -9,7 +10,7 @@ def new
   def create
     @user = User.find(params[:user_id])
     @post = Post.find(params[:post_id])
-    @comment = @post.comments.new(comment_params.merge(author: @user))
+    @comment = @post.comments.new(comment_params.merge(user: @user))
```

Comment:
- [ ] I cannot create a comment to test the comments authorization. Kindly resolve the error.

  ![image](https://github.com/noelfoka/Blog_app/assets/54848363/70d55086-7686-4cfe-ad5b-609a7be441f6)